packages/openapi-to-graphql/src/index.ts

```diff
@@ -519,7 +517,8 @@ async function translateOpenAPIToGraphQL(
     }, 0)
 
   /**
-   * Organize created queries / mutations / subscriptions into viewer objects.
+   * Organize authenticated Query, Mutation, and Subscriptions fields into
+   * viewer objects.
```

Comment:
I think that quite soon we will see the issue of the auth for Subscription, because for now i think it's a rather limited because it's based on HTTP security schemes that will only apply to Websocket eventually.
Usually for a subscription, the authentication takes place a connection time, from parameter set in the PubSub client instance. But then Pubsub instance like eventEmitter don't take auth parameter...
So those authViews could set the connection params, if it's possible ... As you can see it might require some thoughts and tests.

Reply:
Yes, I see! We should file an issue for this. I can get started on it and if you'd like to add anything, feel free to do so! But for this PR, I think we can go ahead :)